### DISCUSSION TITLE
Remote Desktop Error

### DISCUSSION THREAD

**Discussion by dincerkaan** [zitadel/zitadel#6299] [Q&A] [answered]
Hello, I am running Zitadel via remote desktop. Then when I try to access zitadel from my desktop, I get this error. I can access CockroachDB but I have problem for zitadel how can I solve it?

ID=QUERY-n0wng Message=Instance not found

**Comment by hifabienne** [comment, marked as answer]
Hei @dincerkaan 
Did you have a look at the following documentation: https://zitadel.com/docs/self-hosting/deploy/troubleshooting#instance-not-found
You can find exactly this error with what configuration you need to do.
